nni/compression/pytorch/compressor.py

```diff
@@ -608,24 +626,35 @@ def quant_backward(tensor, grad_output, quant_type):
         tensor
             gradient of the input of quantization operation
         """
```

Comment:
Comment of this function need to be updated or delete.

Reply:
yeah, u r right.... . I'll update it directly.